lapce-data/src/editor.rs

```diff
@@ -284,9 +284,10 @@ impl LapceEditorBufferData {
                     })
                     .collect::<Vec<(lapce_core::selection::Selection, &str)>>()
             });
-        let additioal_edit: Option<Vec<_>> = additional_edit.as_ref().map(|edits| {
-            edits.iter().map(|(selection, c)| (selection, *c)).collect()
-        });
+        let additional_edit: Option<Vec<_>> =
```

Comment:
I know you only fixed the typo here, but can we have the `unwrap_or_default` call here, please?